docs/src/modules/components/ChartComponentsGrid.js

```diff
@@ -71,23 +122,50 @@ export default function ChartComponentsGrid() {
               sx={(theme) => ({
```

Comment:
I made this loading="lazy" in e9b209f6e3f77c9df6ed68c9195fb62289119b6b

![Screenshot 2024-02-20 at 01 16 28](https://github.com/mui/mui-x/assets/3165635/618bf9d6-1f84-450f-8376-ea037992e92b)

https://pagespeed.web.dev/analysis/https-next-mui-com-x-react-charts/8yni6fpzlc?form_factor=mobile

I landed here trying to optimize our SEO ranking for "react charts". Google uses performance as part of its indicators, and since it's pretty competitive, any small edge might make the difference.

Reply:
Ah, cool, totally missed this—thank you!